src/Octoshift/GithubApi.cs

```diff
@@ -148,7 +148,7 @@ public virtual async Task<string> GetOrganizationId(string org)
             });
 
             return response.Outcome == OutcomeType.Failure
-                ? throw new OctoshiftCliException("Failed to lookup the Organization ID", response.FinalException)
+                ? throw new OctoshiftCliException($"Failed to lookup the Organization ID for {org}", response.FinalException)
```

Comment:
I think this would be even better if we wrap the slug in quotes, to make it really clear that there should be a value there:

```suggestion
                ? throw new OctoshiftCliException($"Failed to lookup the Organization ID for organization '{org}'", response.FinalException)
```